src/overlays/actors/ovl_Obj_Spidertent/z_obj_spidertent.c

```diff
@@ -28,118 +35,789 @@ const ActorInit Obj_Spidertent_InitVars = {
     (ActorFunc)ObjSpidertent_Draw,
 };
 
-// static ColliderTrisElementInit sTrisElementsInit[6] = {
-static ColliderTrisElementInit D_80B31060[6] = {
+static ColliderTrisElementInit sTrisElementsInit1[6] = {
```

Comment:
```suggestion
static ColliderTrisElementInit sTrisElementsInit1[] = {
```
We try to avoid hard-coded array sizes unless necessary